src/bin/rustfmt.rs

```diff
@@ -90,6 +91,14 @@ impl CliOptions {
             }
         }
 
+        if let Some(ref color) = matches.opt_str("color") {
+            if let Ok(color) = Color::from_str(color) {
```

Comment:
nit: prefer `match` to `if let ... else`

Reply:
I based this from parsing `write-mode` above. Would it be ok to still use `if-let` because it is only 2 branches and both branches have different behaviors?

Reply:
It's fine to leave, in general, its always better to use `match` rather than `if let ... else` since internally they are the same and `match` is more succinct, easier to change, and more common in Rust.